frontend/src/pages/HomePage.css

```
@@ -0,0 +1,366 @@
+  margin: 0 auto;
+  padding: 28px 16px 72px;
+}
+
+.wgp-home__hero {
+  padding: 28px 8px 8px;
+  text-align: center;
+  animation: wgp-home-rise 0.55s ease both;
+}
+
+.wgp-home__brand {
+  font-family: 'Fraunces', Georgia, serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Fraunces" (font-family-name-quotes)

(font-family-name-quotes)